test/functional/JIT_Test/src/jit/test/jitserver/JITServerTest.java

```diff
@@ -662,8 +665,11 @@ public void testServerAOTCache() throws IOException, InterruptedException {
 			destroyCache("test_jitscc");
 			Thread.sleep(DESTROY_SCC_WAIT_TIME_MS);
 
-			if(!checkLogFiles("testServerAOTCache.client"+randomLogId+".jitverboselog.out.*", "remote deserialized"))
+			if(!checkLogFiles("testServerAOTCache.client"+randomLogId+".jitverboselog.out.*", "remote deserialized")) {
+				logger.info("No deserialized methods found at the client. Stopping server...");
+				destroyAndCheckProcess(server, serverBuilder);
```

Comment:
I don't understand how destroying the server here will allow us to determine what the problem was.
I thought the assumption was that the server is not alive when the client starts for the second time and that's the reason the client doesn't find any deserialized methods.

Reply:
It's not much, but `destroyAndCheckProcess` will report the exit code of the server process and dump its logs if the code was abnormal. That will at least confirm that the problem is a server crash.